File: 백준/Bronze/5586. JOI와 IOI/JOI와 IOI.java
import java.io.*;
import java.util.*;

public class Main {
    public static void main(String[] args) throws IOException {
        Scanner sc = new Scanner(System.in);
        String JOIOI = sc.next();
        sc.close();
        int JOIcount = 0;
        int IOIcount = 0;
        for (int i = 0; i < JOIOI.length() - 2; i++) {
            String check = String.valueOf(JOIOI.charAt(i)) + String.valueOf(JOIOI.charAt(i + 1))
                    + String.valueOf(JOIOI.charAt(i + 2));
            if (check.equals("JOI")) {
                JOIcount++;
            } else if (check.equals("IOI")) {
                IOIcount++;
            }
        }
        System.out.println(JOIcount);
        System.out.println(IOIcount);
    }
}

File: 백준/Bronze/5586. JOI와 IOI/JOI와 IOI.py
JOIOI = input()

JOIcount = 0
IOIcount = 0
for i in range(len(JOIOI)-2):
    if JOIOI[i:i+3] == "JOI":
        JOIcount += 1
    elif JOIOI[i:i+3] == "IOI":
        IOIcount += 1

print(f"{JOIcount}\n{IOIcount}")